enhancements/audit-command.md

```diff
@@ -0,0 +1,722 @@
+
+This command will perform checks by using the [SDK](https://github.com/operator-framework/operator-sdk) available sub commands; `bundle validate` and `scorecard bundle`.
+
+In this way, the binary will provide the alpha audit sub-command which would like: 
+
+```
+$ alpha audit --help
+
+Flags:
+  -h, --help                     help for audit
+  -i, --index-image              index image to get package from. 
+  -c, --container-tools          the container tool used to login in the registry and pull and extract the images. One of: [docker, podman] (default "docker")
+  -w, --wait-time duration       minutes to wait for the report to be complete. Example: 60m (default timeout is 30m)       
+  -o, --output                   output format for results. Valid values: xlsx, json (default "xlsx")
+  -s, --scorecard bool           set false to disable the scorecard checks. (default true). Note that scorecard test requires a cluster running. 
+  -v, --validation bool          set false to disable the bundle validation checks. (default true). 
+  -d, --data                     set false to disable the data report. It is useful for who would like to audit the index via only run the bundle and scorecard checks. (default true).
+  -h, --head-only                set the flag to audit only the operator bundles which are the head of the channel. (default all bundles of the index catalog image will be used)
+  -w, --warning                  set false to disable the report of warning(s) found in the checks (default warning and errors will be reported)
+```
+
+Note that the command will:
+- Use the index image to gathering the data
+- Download the operator bundles
+- Use SDK commands to run the checks 
+- Output the results
+
+**NOTE** If we move forward with this command as part of opm it would be like: `opm alpha index audit --help`
+ 
+### User Stories 
+ 
+- As an admin and or maintainer of OLM index catalog, I would like to check if all bundle operators in the catalogue are respecting all expected criteria so that, I can ensure a better quality of all bundles in the catalogue.
+- As an admin and or maintainer of OLM index catalog, I would like to easily know more about the bundles in the catalogue so that, I can easily provide strategies which will be useful for the biggest part of the authors of the projects published on it.
+- As an admin and or maintainer of OLM index catalog, identify problems and concerns in the operators bundles of some catalogue so that, I can easily decide if they should or not be removed from the index catalog.
+- As an admin and or maintainer of OLM index catalog I would like to audit the OLM index catalog by running custom tests for each operator bundle that, I can use it to verify if all operator bundles will work as expected.
```

Comment:
This really solidifies to me that this should go into `opm` and NOT `operator-sdk` because it is the admin persona that is more relevant.

Reply:
I think it was answered in other similar comments so marking it as solved.